src/ngx_http_upsync_module.c

```diff
@@ -3676,6 +3676,40 @@ ngx_http_upsync_set(ngx_conf_t *cf, ngx_command_t *cmd, void *conf)
     return NGX_CONF_OK;
 }
 
+static void ngx_http_upsync_show_upstream(ngx_http_upstream_srv_conf_t *uscf, ngx_buf_t *b) {
+    ngx_http_upstream_rr_peers_t    *peers = NULL;
+    ngx_uint_t                       i;
+    ngx_str_t                       *host;
+
+    host = &(uscf->host);
+
+    if (uscf->peer.data != NULL) {
+        peers = (ngx_http_upstream_rr_peers_t *) uscf->peer.data;
+    }
+
+    b->last = ngx_snprintf(b->last, b->end - b->last, "Upstream name: %V; ",
+            host);
+    b->last = ngx_snprintf(b->last, b->end - b->last,
+            "Backend server count: %d\n", peers->number);
+
+    for (i = 0; i < peers->number; i++) {
+        b->last = ngx_snprintf(b->last, b->end - b->last, "        server %V",
+                &peers->peer[i].name);
```

Comment:
Nginx's style is: 
b->last = ngx_snprintf(b->last, b->end - b->last, "        server %V",
                                    &peers->peer[i].name);